### ISSUE TITLE
Can I enable RFP without the keyboard changes?

### ISSUE THREAD

**Issue by infokiller** [arkenfox/user.js#1036]
Hi and many thanks for all the work you put in this project.

When I enable RFP by setting `privacy.resistFingerprinting`, it breaks Alt keybindings that I define in the "Manage Extension Shortcuts" page (which use the web extensions commands API).
The user.js file has the following relevant comment about this:

https://github.com/arkenfox/user.js/blob/c367beabe38750dfc21cd8c5da27239ed0529845/user.js#L1449-L1452

I also found relevant issues:

- https://github.com/arkenfox/user.js/issues/391#issuecomment-486137363
- https://github.com/tridactyl/tridactyl/issues/1500
- <https://bugzilla.mozilla.org/show_bug.cgi?id=1536533>: this bug is marked as resolved although the issue is still there for me

However, I haven't yet found a solution or workaround.
The simplest "workaround" would be to just enable RFP except the keyboard handling stuff. Is that possible?

I'm using Firefox 81.0.1

## ~~Steps to reproduce~~ update: wasn't able to reproduce this in a new profile :(the tridactyl issue is still reproducible for me though)

- Install [Foxy Calculator](https://addons.mozilla.org/en-US/firefox/addon/foxy-calculator/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)
- Set the "Activate toolbar button" keybinding to `Alt-c` using the "Manage Extension Shortcuts" page
- Press `Alt-c`

Thanks!

**Comment by Thorin-Oakenpants** [comment]
```
 ** 1222285 & 1433592 - spoof keyboard events and suppress keyboard modifier events (FF59+)
      Spoofing mimics the content language of the document. Currently it only supports en-US.
      Modifier events suppressed are SHIFT and both ALT keys. Chrome is not affected.
      FF60: Fix keydown/keyup events (1438795)
```

1536533 is marked as INVALID. not resolved

RFP is an all-in or nothing. You can't disable the shift+alt key event spoofing. 

RFP wants to do two things
- one: tighten up what happens (all-in or nothing) so extensions and prefs can't affect the anti-FPing
   - e.g. in canvas, by default, an extension never even sees the canvas to modify it
   - e.g. changing the font visibility pref does not affect how RFP reads it
- two: reduce breakage
   - there's these things called [principals](https://blog.mozilla.org/attack-and-defense/2020/06/10/understanding-web-security-checks-in-firefox-part-1/)
   - not all principals need RFP applied (and many don't in various functions etc)
   - e.g spoofing as FF78 doesn't mean telemetry reports 78
   - your issue is with RFP affecting extensions: and in general, extensions don't need to be lied to: but I guess it depends on what is being protected and if the web extension could leak that: e.g. you're protected as UTC 0, but the script injects into web content and leaks that you're really on Easter Island (or Christmas Island.. or SKULL ISLAND)
   - [1635603](https://bugzilla.mozilla.org/show_bug.cgi?id=1635603) is what you need to keep an eye on

**Comment by infokiller** [comment]
Thanks for the quick response! I'll dig into these links.
